src/parseExpensiMark.ts

```diff
@@ -292,4 +292,4 @@ function parseExpensiMark(markdown: string): MarkdownRange[] {
   return groupedRanges;
 }
 
-export default parseExpensiMark;
+export {parseExpensiMark, sortRanges};
```

Comment:
I don't think it's a good idea to export and use `sortRanges` from `parseExpensiMark`. This is a self-contained parser used as default which we plan to remove from the repo at some point so we shouldn't use any of its pieces in the library code. Instead, let's extract `sortRanges` to a separate file (maybe something like `parserUtils.ts`?) and import it here from there.

Reply:
Okay, I moved them to the `rangeUtils` since `parserUtils.ts` already exists on the web